dtfs-central-api/api-tests/api.js

```diff
@@ -56,4 +56,80 @@ module.exports = (app) => ({
         .delete(url)
         .send(data),
   }),
+  as: (user) => {
+    const token = (user && user.token) ? user.token : '';
+
+    return {
+      post: (data) => ({
+        to: async (url) => request(app)
+          .post(url)
+          .set({ Authorization: token || '' })
+          .send(data),
+      }),
+
+      postMultipartForm: (data, files = []) => ({
+        to: async (url) => {
+          const apiRequest = request(app)
+            .post(url)
+            .set({ Authorization: token || '' });
+
+          if (files.length) {
+            files.forEach((file) => apiRequest.attach(file.fieldname, file.filepath));
+          }
+
+          Object.entries(data).forEach(([fieldname, value]) => {
+            apiRequest.field(fieldname, value);
+          });
+
+          return apiRequest;
+        },
+      }),
+
+      postEach: (list) => ({
```

Comment:
are we using all of these functions? if we only need 1/2, can we keep only those and remove everything else?